frontend/pages/manage-account.tsx

```diff
@@ -0,0 +1,112 @@
+  const [password, setPassword] = useState('')
+  const [retypedPassword, setRetypedPassword] = useState('')
+
+  const updateUserName = (event: SyntheticEvent) => { setUserName((event.target as HTMLInputElement).value) }
+  const updatePassword = (event: SyntheticEvent) => { setPassword((event.target as HTMLInputElement).value) }
+  const updateRetypedPassword = (event: SyntheticEvent) => { setRetypedPassword((event.target as HTMLInputElement).value) }
+
+  const handleSubmit = () => {
+    fetch(`/api/user/:${userID}`, {
+      method: 'PATCH',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({ password })
+    }).catch((error) => {
+      console.log(`Error caught on update submit event: ${error}`)
+    })
+  }
+
+  // Will need username and ID (per the database) stored in state
+  // This will either be passed into this function via props OR we will need a fetch call in this component
+
+  interface deleteAccountResponse {
+    status: string
+  }
+
+  const handleDeleteAccount = () => {
+    setWaiting(true)
+    fetch(`/api/user/:${userID}`, {
+      method: 'DELETE'
+    }).then((data): Promise<deleteAccountResponse> => {
+      return data.json() as Promise<deleteAccountResponse>
+    }).then((responseBody: deleteAccountResponse) => {
+      console.log(responseBody.status)
+      if (responseBody.status) { updateModalForDelete() } else { throw new Error('Problem with respone from delete handler. See server logs.') }
+    }).catch((error) => {
+      console.log(`error caught on delete event: ${error}`)
+      updateModalForDelete(error)
+    })
+  }
+
+  const updateModalForDelete = (error = false) => {
+    const deleteAccountModal = document.querySelector('#modalBox-deleteAccount')
+    if (deleteAccountModal) {
+      setWaiting(false)
+      deleteAccountModal.innerHTML = (error ? `Operation failed: ${error}` : 'Operation was successful. <br /> What\'s done is done.') + '<br /> Redirecting to the home page in 5 seconds'
+      setTimeout(() => {
+        window.location.href = '/'
+      }, 5000)
+    }
+  }
+
+  return (
+    <div id="create-account" className=" flex flex-col justify-center items-center min-w-max">
+      <header className="mt-2 mb-7 text-2xl">This is a Header Stand-in for Account Management </header>
+      <p className="text-lg"> Welcome to the account management console </p>
+      <p> From this page, you can update your password or delete your account (please don't!) </p>
+      <div className='border p-4 w-full mt-7 max-w-xs rounded-md'>
+
+        <div className="form-control w-full max-w-xs">
+          <label className="label">
+            <span className="label-text">E-mail</span>
+            <a href="#modal-deleteAccount"><span className="label-text text-muted"> delete account </span></a>
+          </label>
+          <input id="Inp-userName" type="text" placeholder="Type here" className="input input-bordered w-full max-w-xs" disabled />
```

Comment:
I see the 'email' field on the frontend that has 'Type here...' placeholder text, but the field itself is locked. I think we should have it unlocked if we want the users to update their email, or remove the 'Type here...' placeholder if it should be locked.

Reply:
Sounds like the props are going to be passed in, so this should not be visible in the future.